packages/matplotlib/test_matplotlib.py

```diff
@@ -113,7 +113,7 @@ def test_svg(selenium):
     plt.savefig(fd, format="svg")
 
     content = fd.getvalue().decode("utf8")
-    assert len(content) == 14998
+    assert len(content) == 15016
```

Comment:
This is not a good test. But we didn't make it any worse in this PR.

Reply:
Agreed, should I remove it altogether?

Reply:
Sorry, I introduced that bad test. Let's remove/update the tests altogether in the separate Pr.